de.fhdo.lemma.operationdsl.metamodel/model/OperationViewpointModel.xcore

```diff
@@ -258,4 +258,16 @@ class ImportedOperationAspect {
 class PossiblyImportedOperationNode {
     refers Import[0..1] ^import
     refers OperationNode node
+
+    op String getTechnologyNames(OperationNode node) {
```

Comment:
Why is the name of the method in plural if it only returns a single name? Moreover, please at a brief describing comment.